pkg/install/install.go

```diff
@@ -47,7 +47,7 @@ type Install struct {
 }
 
 const (
-	everestBackendServiceName = "percona-everest-backend"
+	everestBackendServiceName = "everest"
```

Comment:
This was causing [the restart problem](https://github.com/percona/everest/actions/runs/8108835236/job/22163432926) - when everest is already installed we were trying to restart the pods with the label `app.kubernetes.io/name: percona-everest-backend` which doesn't exist anymore.

in Beta we still use the [quickstart-k8s.yaml](https://github.com/percona/percona-everest-backend/blob/main/deploy/quickstart-k8s.yaml) from the https://github.com/percona/percona-everest-backend main branch, where we use that old labels for the resources, that why the problem didn't show up before we actually started to use quickstart-k8s.yaml from the monorepo with the new labels